torchvision/ops/misc.py

```diff
@@ -70,20 +70,23 @@ def __init__(
         self,
         in_channels: int,
         out_channels: int,
-        kernel_size: int = 3,
-        stride: int = 1,
-        padding: Optional[int] = None,
+        kernel_size: Union[int, Tuple[int, ...]] = 3,
```

Comment:
I was scared that by adding Union are we JIT compatible? I had avoided Union for the same.

Reply:
thanks for the input @oke-aditya , let me check if this is JIT compatible or not. Will update here

Reply:
There are no JIT-scriptability concerns here. Constructors can have whatever calls. It's on the forward calls that we have the restrictions. :)

Reply:
I have also try to do 
```
import torchvision.models as models
import torch

m = models.efficientnet_b0()
torch.jit.script(m)
```
and it works as well (notes: efficientnet use Conv2dNormActivation)